odoo/addons/base/models/ir_mail_server.py

```diff
@@ -275,17 +303,28 @@ def test_smtp_connection(self):
                     # ignored, just a consequence of the previous exception
                     pass
 
-        message = _("Connection Test Successful!")
+        messages = [_('Connection Test Successful!')]
+        if autodetect_max_email_size:
+            if self.max_email_size:
```

Comment:
Looks like this method is meant to be used in multi-mode so this is not correct. 
Not sure what to do with the messages if that's the case... do we really need a message? ^^